src/moin/apps/frontend/views.py

```diff
@@ -898,8 +899,15 @@ def ajaxdelete(item_name):
         for itemname in itemnames:
             response["itemnames"].append(itemname)
             itemname = subitem_prefix + itemname
+            if '%' in itemname:
```

Comment:
if it is correct with that "if" it should work in the same way even if the condition is not true.